json/src/main/java/org/jdbi/v3/json/internal/StubJsonMapper.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.json.internal;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+import org.jdbi.v3.core.internal.exceptions.Unchecked;
+import org.jdbi.v3.core.result.UnableToProduceResultException;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.jdbi.v3.core.statement.UnableToCreateStatementException;
+import org.jdbi.v3.json.JsonConfig;
+import org.jdbi.v3.json.JsonMapper;
+
+public class StubJsonMapper implements JsonMapper {
+    private static final String NO_IMPL_INSTALLED = String.format(
+        "you need to install (%s.%s) a %s impl, like jdbi3-jackson2 or jdbi3-gson2",
+        JsonConfig.class.getSimpleName(),
+        Unchecked.<String, Class[], Method>biFunction(JsonConfig.class::getMethod).apply("setJsonMapper", new Class[] {JsonMapper.class}).getName(),
```

Comment:
Doesn't this simply evaluate to `setJsonMapper`?  This invocation looks crazy complicated and involves reflection (???)

Reply:
I know this would come up :)

This will cause an exception at the start of runtime (e.g. during a test) if `setJsonMapper` is ever renamed. Just putting setJsonMapper in there will allow it to go out of sync. Same reason I always use `.class` references to almost every Type I mentioned in exception strings.

Reply:
People could roll their own JsonMapper, like jaxb

Reply:
Thanks for changing this.  I appreciate where you're coming from but let's KISS.  Exactly how to implement it is much better served as a Javadoc comment rather than a runtime error message.